model/export/SingleDeliveryResultsExporter.php

```diff
@@ -136,7 +136,8 @@ public function getColumnsToExport(): array
                 $columns = array_merge(
                     $this->columnsProvider->getTestTakerColumns(),
                     $this->columnsProvider->getDeliveryColumns(),
-                    $variables
+                    $variables,
+                    $this->columnsProvider->getDeliveryExecutionColumns()
```

Comment:
Just to be clear, array_merge overwrite array keys with same names so the last one are preserved. Variable $variables (duh!) previously was last one and now it isn't and that could have some repercussions. We should watch out for that but probably that's nothing.

Reply:
Just left a comment in slack about UI export feature behavior that might be affected by this change